personnel.md

```diff
@@ -0,0 +1,216 @@
+| [PPI Framework]                   | Paul Dragoonis ([@dr4goonis])         |
+| [PrestaShop]                      | Rémi Gaillard ([@xtaz07])             |
+| [PyroCMS]                         | Ryan Thompson ([@RyanThePyro])        |
+| [ReactPHP]                        | Cees-Jan Kiewiet ([@WyriHaximus])     |
+| [Revive Adserver]                 | Matteo Beccati ([@mbeccati])          |
+| [Sculpin]                         | Chris Tankersley ([@dragonmantank])   |
+| [Slim]                            | Jason Coward ([@drumshaman])          |
+| [SilverStripe]                    | Damian Mooyman                        |
+| [Stash]                           | Robert Hafner ([@tedivm])             |
+| [Stormpath PHP SDK]               | Brian Retterer ([@bretterer])         |
+| [SugarCRM]                        | Andreas Sandberg ([@yellowandy])      |
+| [Symfony]                         | Fabien Potencier ([@fabpot])          |
+| [Flow] and [Neos]                 | Karsten Dambekalns ([@kdambekalns])   |
+| [Wikibase] and [Semantic Media]   | Jeroen De Dauw ([@JeroenDeDauw])      |
+| [Yii framework]                   | Alexander Makarov ([@sam_dark])       |
+| [Zend Framework 2]                | Matthew Weier O'Phinney ([@mwop])     |
+| [Zikula]                          | Shefik Macauley ([@shefik_info])      |
+
+
+## Past
+
+### Former Secretaries
+
+| Name                                  | Term                    |
+|---------------------------------------|-------------------------|
+| Amanda Folsom ([@ambassadorawsum])    | 2016-08-28 - 2017-11-12 |
+| Samantha Quiñones ([@ieatkillerbees]) | 2016-02-28 - 2016-12-24 |
+| Joe Ferguson ([@joepferguson])        | 2016-01-31 - 2016-06-29 |
+
+
+### Former Core Committee Members
+
+| Name                                  | Term                    |
+|---------------------------------------|-------------------------|
+| TBD                                   |                         |
+
+### Former Member Projects
+
+| Project                                   | Representative                        |
+|-------------------------------------------|---------------------------------------|
+| [Aura] and [Solar]                        | Paul M. Jones ([@pmjones])            |
+| [Assetic]                                 | Kris Wallsmith ([@kriswallsmith])     |
+| [Agavi]                                   | David Zülke ([@dzuelke])              |
+| [Doctrine]                                | Guilherme Blanco ([@guilhermeblanco]) |
+| [Guzzle]                                  | Jeremy Lindblom ([@jeremeamia])       |
+| [Laravel]                                 | Taylor Otwell ([@taylorotwell])       |
+| [The League of Extraordinary Packages]    | Graham Hoefer ([@greydnls])           |
+| [Propel]                                  | William Durand ([@couac])             |
+| [sabre/dav]                               | Evert Pot ([@evertp])                 |
```

Comment:
There are a lot more than this, I think we just shouldn't list former projects.

Reply:
Hmm, I browsed the whole git history of `members.html`, I'll check again to be sure.

Anyway, I think it has some value to keep an historical record of who's been part of the group, if anything because we wouldn't otherwise mention people who gave valuable contributions in the past.

Reply:
I'm fine with listing ex-projects for historical reasons, but I it's not a release blocker.